js/views/userPage/UserPage.js

```diff
@@ -132,17 +136,29 @@ export default class extends baseVw {
   }
 
   createFollowersTabView(opts = {}) {
-    return this.createChild(this.tabViews.Follow, {
+    const newView = this.createChild(this.tabViews.Follow, {
       ...opts,
       followType: 'Followers',
     });
+
+    newView.on('updateCount', (count) => {
+      this.updateFollowerCount(count);
+    });
+
+    return newView;
   }
 
   createFollowingTabView(opts = {}) {
-    return this.createChild(this.tabViews.Follow, {
+    const newView = this.createChild(this.tabViews.Follow, {
       ...opts,
       followType: 'Following',
     });
+
+    newView.on('updateCount', (count) => {
+      this.updateFollowingCount(count);
```

Comment:
If we do decide to leave this event, can you only fire it if the count actually changes?